src/main/java/org/jitsi/jigasi/transcription/TranslationManager.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Jigasi, the JItsi GAteway to SIP.
+ *
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jigasi.transcription;
+
+import java.util.*;
+
+/**
+ * This class manages the translations to be done by the transcriber.
+ *
+ * @author Praveen Kumar Gupta
+ */
+public class TranslationManager
+    implements TranscriptionListener
+{
+
+    /**
+     * List of target languages for translating the transcriptions.
+     */
+    private Map<String, Integer> languages = new HashMap<>();
```

Comment:
I don't think this HashMap is threadsafe. Can you make it so calls to the map are synchronised?